.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
   pull_request:
 
 env:
-  docker_repository: docker.synapse.org/syn25992884/bert-phi-annotator
+  docker_repository: docker.synapse.org/syn22277123/phi-annotator-huggingface-bert
```

Comment:
Actually, we might use this one GitHub repo to publish many Docker images from HuggingFace. So instead of `phi-annotator-huggingface-bert`, how about just `phi-annotator-huggingface`

Reply:
Sounds good! I'll change this. Note that this is the root name of the image and that a suffix will be added where the suffix is the "config name" used to represent a model.

Reply:
I'll also rename this repository from `phi-annotator-huggingface-bert` to `phi-annotator-huggingface`